pkg/pebinaryexecutor/pebinary.go

```diff
@@ -67,6 +67,8 @@ const (
 	CNICalico = "calico"
 	CNICilium = "cilium"
 	CNICanal  = "canal"
+	LogPath   = "C:\\var\\log\\"
```

Comment:
The standard path for linux is `/var/lib/rancher/rke2/agent/logs/kubelet.log`, why do we use `c:/var/log` on windows instead of `c:/var/lib/rancher/rke2/agent/logs` ?

Reply:
AH! I missed that already existing path! Thanks Brad! I'm actually going to move calico logs to there as well, I think it makes much more sense